src/Tools/dotnet-monitor/CollectionRules/Actions/CollectDumpAction.cs

```diff
@@ -70,7 +70,7 @@ public CollectDumpAction(IServiceProvider serviceProvider, IEndpointInfo endpoin
                     EgressOperation egressOperation = new EgressOperation(
                         token => {
                             startCompletionSource.TrySetResult(null);
-                            return _dumpService.DumpAsync(EndpointInfo, dumpType, token);
+                            return _dumpService.DumpAsync(EndpointInfo, dumpType, PackageMode.None, token);
```

Comment:
Is this something we're going to want to expose as an option on the CollectDumpOptions at some point? Don't have to do it now.

Reply:
I think we should long term, but I was not going to make this a part of this change.